pages/common/pwsh.md

```diff
@@ -1,20 +1,25 @@
 # pwsh
 
 > PowerShell Core is a cross-platform automation and configuration tool/framework.
+> See also: `powershell`.
 > More information: <https://learn.microsoft.com/powershell/>.
 
-- Start an instance of PowerShell:
+- Start an interactive shell session:
 
 `pwsh`
 
-- Execute a script and then exit:
+- Start an interactive shell session without loading startup configs:
 
-`pwsh -File {{path/to/file.ps1}}`
+`pwsh -NoProfile`
 
-- Set the execution policy for the current session:
+- Execute specific commands:
 
-`pwsh -ExecutionPolicy {{AllSigned|Bypass|Default|RemoteSigned|Restricted|Undefined|Unrestricted}}`
+`pwsh -Command "{{echo Hello world}}"`
```

Comment:
```suggestion
`pwsh -Command "{{string}}"`
```
Maybe we can make it a generic placeholder.

Reply:
Generic placeholders are the only one reliable way (among placeholders) to present smth without confusion IMO. :)